msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -233,6 +243,12 @@ Upon first run of local debug to Office or Outlook, you'll be prompted to sign i
 
 Provide feedback and report any issues with the Teams Toolkit debugging experience at [Microsoft Teams Framework (TeamsFx)](https://github.com/OfficeDev/TeamsFx/issues).
 
+#### Mobile debugging
+
+1. You can remotely debug your app in Office for Android. From your profile **(Me) > Settings > Allow debugging**, toggle on the option for **Enable remote debugging**.
+1. From your dev machine, open your browser to the DevTools inspection page. For example with Microsoft Edge, open `edge://inspect/#devices` to display a list of debug-enabled Android WebViews. The WebView running your tab application in Office for Android is [placeholder].
```

Comment:
For example, go to `edge://inspect/#devices` in Microsoft Edge to display a list of debug-enabled Android Webviews.